package.json

```diff
@@ -191,7 +191,8 @@
       "node-menu",
       "metrics-cluster-feature",
       "license-menu-item",
-      "support-page"
+      "support-page",
+      "example-extension"
```

Comment:
no need to enable `example-extension` for all.

Reply:
I'll try find another way to force it's build failure to block PRs. Since I actually suspect that no extension build failure blocks PRs at the moment. 

However, I do think that we should (at least for testing) always have it activated because we should be constantly testing it to make sure that it doesn't start throwing errors.

Reply:
If we want to have it tested automatically then we need to come up with a way to inject it during tests.

Reply:
okay sounds good